Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-publicpage.jsp

```diff
@@ -0,0 +1,25 @@
+<%@ taglib uri="http://tiles.apache.org/tags-tiles" prefix="tiles"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>

```

Comment:
this file was renamed. Please delete this and use the correct one (should be name "nav-settings")